cleanlab/rank.py

```diff
@@ -332,14 +372,20 @@ def get_label_quality_ensemble_scores(
     elif weight_ensemble_members_by == "accuracy":
         weights = np.array(accuracy_list) / sum(accuracy_list)  # Weight by relative accuracy
         if verbose:
-            print("Ensemble members will be weighted by: their relative accuracy")
+            print("Ensemble members will be weighted by their relative accuracy")
             for i, acc in enumerate(accuracy_list):
                 print(f"  Model {i} accuracy : {acc}")
                 print(f"  Model {i} weights  : {weights[i]}")
 
         # Aggregate scores with weighted average
         label_quality_scores = (scores_ensemble * weights).sum(axis=1)
 
+    elif weight_ensemble_members_by == "log_loss_search":
+        weights = neg_log_loss_weights  # Weight by exp(t * -log_loss) where t is found by searching set of T
+
```

Comment:
also print the weights here if verbose?

More generally can probably move lines 376-381 to make them apply to both weight_ensemble_members_by = 'accuracy' and 'log_loss_search' without copy/pasting code.

Reply:
Sounds good. Added commit to print weights for `"log_loss_search"` if verbose.

Agree, can refactor in the future to remove some redundancies.